package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
-  "version": "2.0.0",
-  "description": "Nemo Plugin to Use selenium-webdriver to take a screenshot",
-  "registerAs": "screenshot",
-  "main": "index.js",
-  "scripts": {
-    "test": "grunt"
-  },
-  "devDependencies": {
-    "chai": "~1.6.0",
-    "grunt": "~0.4.1",
-    "grunt-contrib-jshint": "~0.7.1",
-    "grunt-simple-mocha": "~0.4.0",
-    "mocha": "~1.10.0",
-    "nemo": "^2.0.0"
-  },
-  "peerDependencies": {
-    "nemo": "^2.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/paypal/nemo-screenshot.git"
-  },
-  "keywords": [
-    "Nemo",
-    "plugin",
-    "screenshot",
-    "selenium",
-    "selenium-webdriver"
-  ],
-  "author": "Matt Edelman",
-  "contributors": [
-    {
-      "name": "Pranav M. Pranav",
-      "email": "pmpranav@paypalcorp.com"
-    }
-  ],
-  "dependencies": {
-    "mkdirp": "~0.5.1"
-  }
+    "name": "nemo-screenshot",
+    "version": "2.0.0",
+    "description": "Nemo Plugin to Use selenium-webdriver to take a screenshot",
+    "registerAs": "screenshot",
+    "main": "index.js",
+    "scripts": {
+        "test": "grunt"
+    },
+    "devDependencies": {
+        "chai": "^3.5.0",
+        "grunt": "^1.0.1",
+        "grunt-contrib-jshint": "^1.0.0",
+        "grunt-simple-mocha": "^0.4.1",
+        "mocha": "^2.4.5",
+        "nemo": "^2.1.1"
+    },
+    "peerDependencies": {
+        "nemo": "^2.1.1"
```

Comment:
Nemo _should_ still be returned. If it's not, that's a bug. The 2.1.1 change is not meant to be "breaking" but it would be if that's the case. Will review.